docs_source_files/content/getting_started/quick.en.md

```diff
@@ -28,7 +28,8 @@ that element. This is not only a time-saver but also an excellent way
 of learning Selenium script syntax.
 
 ## Grid
+Selenium Grid is a feature in Selenium that allows you to run test cases in different machines across different platforms. The control of triggering the test cases is on the local machine, and when the test cases are triggered, they are automatically executed by the remote machine
```

Comment:
Hi @rahuljhakant ,

Thanks for the PR. Can you please make sure to have a line width of 70-80

Thanks,
Harsha